internal/cmd/profile/login_command.go

```diff
@@ -211,3 +213,39 @@ func persistAccessCredentials(creds *session.StoredCredentials) error {
 		Credentials: creds,
 	})
 }
+
+func printEnvWarning() {
+	envVars := checkEnvironmentVariables()
+	if len(envVars) == 0 {
+		return
+	}
+
+	fmt.Println("WARNING: The following Spacelift environment variables are set:")
+	for _, envVar := range envVars {
+		fmt.Printf("   - %s\n", envVar)
+	}
+	fmt.Println("\nEnvironment variables take precedence over profile credentials.")
+	fmt.Println("If these credentials are expired or invalid, commands will fail even after successful login.")
+}
+
+// checkEnvironmentVariables checks for Spacelift environment variables and returns a list of those that are set.
+func checkEnvironmentVariables() []string {
+	var envVars []string
+
+	checkVars := []string{
+		session.EnvSpaceliftAPIToken,
+		session.EnvSpaceliftAPIKeyEndpoint,
+		session.EnvSpaceliftAPIEndpoint,
+		session.EnvSpaceliftAPIGitHubToken,
+		session.EnvSpaceliftAPIKeyID,
+		session.EnvSpaceliftAPIKeySecret,
```

Comment:
Aren't we going to print secret credentials to stdout with that? 🤔

Reply:
That'd be funny.

We just print the name of the var:
```
WARNING: The following Spacelift environment variables are set:
   - SPACELIFT_API_ENDPOINT

Environment variables take precedence over profile credentials.
If these credentials are expired or invalid, commands will fail even after successful login.
```